charts/guac/values.yaml

```diff
@@ -233,6 +244,25 @@ guac:
     #     memory: "0.5G"
     dayBetweenRescan: 0
 
+  cdCertifier:
+    enabled: true
+    name: cd-certifier
+    annotations:
+      reloader.stakater.com/auto: "true"
+    replicas: 1
+    image:
+      command: ['sh', '-c', '/opt/guac/guaccollect cd']
+    env: []
+    nodeSelector: {}
+    tolerations: []
+    serviceAccount:
+      create: true
+      annotations: {}
+    resources: {}
+    #   requests:
+    #     cpu: "0.25"
+    #     memory: "0.5G"
+
```

Comment:
Dont you need to add here the same thing as osv like `dayBetweenRescan: 0`, latency...etc?

Reply:
Moved dayBetweenRescan up to the common section to generally apply to all certifiers